tt_eager/tt_lib/csrc/tt_lib_bindings_tensor_dm_ops.cpp

```diff
@@ -102,6 +102,23 @@ namespace tt::tt_metal::detail{
                         "input", "Input tensor for which repetition is computed", "Tensor", "Tensor of any shape", "Yes"
                         "size", "The number of times to repeat this tensor along each dimension", "List[Int]", "Positive repetition values", "Yes"
                         "output_mem_config", "Layout of tensor in TT Accelerator device memory banks", "MemoryConfig", "Default is interleaved in DRAM", "No"
+                        "output_tensor", "Optional Output Tensor", "Tensor", "Default value is None", "No"
+                )doc");
+
+        m_tensor.def("repeat", py::overload_cast<uint8_t, const Tensor&, const Shape&, const MemoryConfig&, std::optional<Tensor>>(&repeat),
+            py::arg("cq_id").noconvert() = 0, py::arg("input"), py::arg("size"), py::arg("output_mem_config").noconvert() = operation::DEFAULT_OUTPUT_MEMORY_CONFIG, py::arg("output_tensor").noconvert() = std::nullopt, R"doc(
```

Comment:
Please change `cq_id` from a positional to a kw_only argument called `queue_id`.

CC @eyonland